Calculator/Calculator/Model/CalculatorItemQueue.swift

```diff
@@ -5,29 +5,33 @@
 //  Created by Erick on 2023/05/29.
 //
 
-struct CalculatorItemQueue {
-    private var queue = CalculatorItemLinkedList()
+struct CalculatorItemQueue<element: CalculateItem> {
```

Comment:
제네릭의 타입을 선언할 때도 타입이므로 대문자로 시작해야 합니다!
```suggestion
struct CalculatorItemQueue<Element: CalculateItem> {
```

Reply:
`Element`로 리네임 하였습니다!!